### PR TITLE
Increase memory in Linux aarch64 (emulated) builds

### PR DESCRIPTION
**Pull request by sergiitk**
Fix the issue with `Linux aarch64 (emulated)` builds failing with 

```
Expiring Daemon because JVM heap space is exhausted
Daemon will be stopped at the end of the build after running out of JVM memory
```

Failed build example: https://source.cloud.google.com/results/invocations/3ebb925d-cdfd-4db8-a57a-30da641bcfcb

### CODE REVIEW

**Comment by sergiitk** [comment]
Here's the run with the successful build: https://source.cloud.google.com/results/invocations/17f509b7-397e-4fe4-9a35-74c9220d7fcb

The build completed, but a few benchmark tests failed. F.e. `io.grpc.benchmarks.driver.LoadWorkerTest` couldn't start the server `java.lang.AssertionError: Server failed to start at `

Also unexpected (for me) warnings in stderr:

> `Apr 26, 2022 1:08:30 AM io.netty.util.internal.MacAddressUtil defaultMachineId WARNING: Failed to find a usable hardware address from the network interfaces; using random bytes: fe:cf:b1:c5:22:5c:41:23`

@ejona86 do you know if these tests were ever passing? Or if adding more memory could help?

File: buildscripts/run_arm64_tests_in_docker.sh [hunk 1]
```diff
@@ -15,10 +15,11 @@ fi
 # so for the rest of the build we will be using "-PskipCodegen=true"
 # avoid further complicating the build.
 docker run $DOCKER_ARGS --rm=true -v "${grpc_java_dir}":/grpc-java -w /grpc-java \
+  -m 2GB \
```
[the remark this answers is not in the document]
Comment (sergiitk) [reply buildscripts/run_arm64_tests_in_docker.sh]:
```
+ cat /sys/fs/cgroup/memory/memory.limit_in_bytes
9223372036854771712
```

You were right, 9223 PB > 2 GB 😄

**Comment by sergiitk** [comment]
Another run: https://source.cloud.google.com/results/invocations/d6396a6d-8f80-4555-8d8e-88334e64cc98

This time only `LoadWorkerTest.runUnaryBlockingClosedLoop` failed. I'll double java memory and see if this changes anything.

**Comment by sergiitk** [comment]
Latest run: https://source.cloud.google.com/results/invocations/d31ba01f-4e46-4970-ba16-22a8c69c0665

Should we merge this one, and tackle the benchmark test issue separately? Or doesn't make sense to have it without the fix?

File: buildscripts/run_arm64_tests_in_docker.sh [hunk 2]
```diff
@@ -10,15 +10,23 @@ else
   DOCKER_ARGS=
 fi
 
+
+cat <<EOF >> "${grpc_java_dir}/gradle.properties"
+skipAndroid=true
+skipCodegen=true
+org.gradle.jvmargs=-Xms128m -Xmx1024m
```
Comment (ejona86) [inline buildscripts/run_arm64_tests_in_docker.sh:17]: Can you remove the `-Xms` setting? It is probably not needed.

**Review by ejona86 — APPROVED** [review]
I think it is fair to submit this and resolve the failing test separately.

File: buildscripts/kokoro/unix.sh [hunk 3]
```diff
@@ -36,7 +36,6 @@ GRADLE_FLAGS+=" -Pcheckstyle.ignoreFailures=false"
 GRADLE_FLAGS+=" -PfailOnWarnings=true"
 GRADLE_FLAGS+=" -PerrorProne=true"
 GRADLE_FLAGS+=" -PskipAndroid=true"
-GRADLE_FLAGS+=" -Dorg.gradle.parallel=true"
```
Comment (ejona86) [inline buildscripts/kokoro/unix.sh:39]: Did you check whether this had an impact? Looking at the gradle documentation it claims this is defaulting to false. But that is a bit surprising to me. https://docs.gradle.org/current/userguide/build_environment.html

Comment (sergiitk) [reply buildscripts/kokoro/unix.sh:39]: What's the expected impact? Build run time? If yes, I see no change in Linux artifacts - it's 26±3 min.

File: buildscripts/run_arm64_tests_in_docker.sh (hunk 2 above)
Comment (sergiitk) [reply buildscripts/run_arm64_tests_in_docker.sh]: This works, build: https://source.cloud.google.com/results/invocations/3397493e-fa81-4ae0-8190-fedc5e7d2823/targets

File: buildscripts/kokoro/unix.sh (hunk 3 above)
Comment (sergiitk) [reply buildscripts/kokoro/unix.sh:39]: Tried parallel build locally:
- BUILD SUCCESSFUL in 9m 49s 
- BUILD SUCCESSFUL in 6m 50s `--parallel`

I'll undo this change.

**Comment by sergiitk** [comment]
Interesting. In the latest build `CachingRlsLbClientTest.rls_withCustomRlsChannelServiceConfig` failed.

**Pull request merged**